source/Octopus.Server.Client/Model/Endpoints/KubernetesTentacleEndpointConfigurationResource.cs

```diff
@@ -3,13 +3,13 @@
 
 namespace Octopus.Client.Model.Endpoints
 {
-    public abstract class TentacleEndpointConfigurationResource : ITentacleEndpointResource
+    public abstract class KubernetesTentacleEndpointConfigurationResource : ITentacleEndpointResource
```

Comment:
I've renamed this and child classes to include `Kubernetes` because now it's got info to do with the Kubernetes Agent so this class needs to make it clear that it is only for Kubernetes Tentacle.

Reply:
Reverted as not worth it.